### PR TITLE
Fix subscription with new graph-node docker image

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

File: ops/graph-codegen.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ async function codegen (cwd) {
     '--output-dir src/types/',
     subgraphLocation
   ], cwd)
-  console.log(result)
+  // console.log(result)
```
Comment (orenyodfat) [inline ops/graph-codegen.js]: please remove this comment

File: ops/setup-env.js [hunk 2]
```diff
@@ -19,9 +19,9 @@ async function setupenv () {
   await require(`../ops/generate-subgraph`)()
 
   const cwd = subgraphRepo
-  console.log('Calling graph codegen')
+  console.log('Calling graph-codegen')
   result = await require(`../ops/graph-codegen`)(cwd)
-  console.log(result)
+  // console.log(result)
```
Comment (orenyodfat) [inline ops/setup-env.js]: please remove this comment

File: package.json [hunk 3]
```diff
@@ -44,7 +44,7 @@
 		"eslint-plugin-standard": "^4.0.0"
 	},
 	"dependencies": {
-		"@graphprotocol/graph-cli": "0.5.1",
+		"@graphprotocol/graph-cli": "git+https://github.com/graphprotocol/graph-cli.git#jannis/graph-create",
```
Comment (orenyodfat) [inline package.json]: I guess we cannot fetch that from a living branch .

File: docker-compose.yml [hunk 4]
```diff
@@ -9,15 +9,16 @@ services:
...
+      node_http: http://graph-node:8000/subgraphs/name/daostack
       node_ws: http://graph-node:8001/subgraphs/name/daostack
       node_rpc: http://graph-node:8020
       ipfs_host: ipfs
       ipfs_port: 5001
       WAIT_HOSTS: ipfs:5001, graph-node:8020, ganache:8545
 
   graph-node:
-    image: graphprotocol/graph-node:v0.5.0
+    image: graphprotocol/graph-node:latest@sha256:a11adda030fcb4ac42d0c31c58cffaacadfda333fecfb1e361d9ec5a72ba1026
```
Comment (orenyodfat) [inline docker-compose.yml]: is this guaranty a specific version?

Comment (jellegerbrandy) [reply docker-compose.yml]: yes exactly, I think this is the most important part of this PR. We were  basically pulling a new image every time they update theirs, which makes our enviroment unreliable.

File: package.json (hunk 3 above)
Comment (jellegerbrandy) [reply package.json]: upgrading to version 0.6.0 that was released 2 days ago.

**Comment by orenyodfat** [comment]
this address issue #58 . right ?

**Pull request merged**